app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'sebastian.torres@wolox.co'
```

Comment:
Better to avoid mentioned you name here, use secrets y .env to store this variable